impl/src/main/java/org/ehcache/config/CacheConfigurationBuilder.java

```diff
@@ -174,16 +167,16 @@ public boolean hasDefaultExpiry() {
     return otherBuilder;
   }
 
-  public CacheConfigurationBuilder<K, V> withKeyCopier(Copier<K> keyCopier) {
+  public CacheConfigurationBuilder<K, V> withKeyCopier(Copier<? super K> keyCopier) {
```

Comment:
In my understanding, ? super K is needed as you can define Long as the key type but configure a Number key copier.

What did I miss?